source/replication/enable-server-side-multi-site-bucket-replication.rst

```diff
@@ -78,14 +78,14 @@ Replication Requires Versioning
 MinIO relies on the immutability protections provided by versioning to
 synchronize objects as part of replication.
 
-Use the :mc-cmd:`mc version suspend` command to enable versioning for the bucket
+Use the :mc-cmd:`mc version enable` command to enable versioning for the bucket
 across *all* MinIO deployments participating in the multi-site replication
 configuration.
 
 .. code-block:: shell
    :class: copyable
 
-   mc version ALIAS/PATH
+   mc version enable ALIAS/PATH
 
 - Replace :mc-cmd:`ALIAS <mc version ALIAS>` with the
```

Comment:
should this also say mc version enable ALIAS?

Reply:
I made a big pass on the web references, so `mc version` is a single page where `enable|disable|info` are treated as parameters only vs subcommands (since there are no additional parameters for each of those, treating them like subcommands felt unnecessary)

So now `mc version ALIAS` links to `mc version` reference docs. Previously that was not the case.